readme.md

```diff
@@ -31,6 +31,48 @@ This includes some recommendations of things we personally favour of course!_
 
 **Next steps?** If you want to [contribute to dwyl projects](https://github.com/dwyl/contributing), get comfortable with [our stack](https://github.com/dwyl/technology-stack) and **open issues whenever you have questions**!
   
+# Spectacle
+
+
+![Spectacle](https://i.postimg.cc/zBZqm4cQ/Screen-Shot-2015-05-09-at-3-11-50-AM.png)
+
+## Why ?
+
+
+So a lot of you may have lots of open windows at once.
+   You got code running and then you got the documentation for the code and you got Google stuff opened.
+  And then "Stack overflow" to search for errors or doubts.
+  Everything becomes a mess and a nightmare with so many pages open.
+  The simplest way to organize everything is to use the spectacle.
+
+## What ?
+
+   Spectacle is simply an app that helps you managing your windows.
+   Lets you modify whatever you want to do to your windows.
+   Keyboard shortcuts are the fastest way to communicate with your computer.
+   If you want to be super fast and efficient "You need to learn how to use Spectacle".
+   Once you learn some of the shortcuts you will just need to use your "Muscle Memory" to use them.
+
+_Note:_ Spectacle is only for Mac in alternative for windows you have AquaSnap that works the same way.
+
+# Spectacle Installation
```

Comment:
Spectacle installation is a sub-section of **Spectacle** (_above_) so having it as a level 3 sub-heading (`###`) makes sense. 
```suggestion
### Spectacle Installation
```